gapic-generator-java/src/main/java/com/google/api/generator/gapic/composer/common/AbstractTransportServiceStubClassComposer.java

```diff
@@ -1361,7 +1364,7 @@ private void createRequestParamsExtractorBodyForRoutingHeaders(
     for (int i = 0; i < routingHeaderParams.size(); i++) {
       RoutingHeaderRule.RoutingHeaderParam routingHeaderParam = routingHeaderParams.get(i);
       MethodInvocationExpr requestFieldGetterExpr =
-          createRequestFieldGetterExpr(requestVarExpr, routingHeaderParam.fieldName());
+          createRequestFieldGetterExpr(requestVarExpr, routingHeaderParam.fieldName(), false);
```

Comment:
Could you a quick comment above to explain why this is always set to false?

Reply:
Because currently explicit routing header parameters are implemented as strings by design. Hence passing false for them, they are not enums.

Reply:
Added as a comment.